src/target/riscv/riscv-013.c

```diff
@@ -5435,6 +5441,8 @@ static int riscv013_step_or_resume_current_hart(struct target *target,
 	if (riscv_reg_flush_all(target) != ERROR_OK)
```

Comment:
If you like:

```suggestion
	if (riscv_reg_cache_flush_all(target) != ERROR_OK)
```

Reply:
I'd like to leave the `flush` as-is. The reason is: while invalidation and `_any_dirty` are only concerned with the cache and do not touch the target, "flush" does.

Reply:
Ok, sure.